src/qml/pages/stub.qml

```diff
@@ -5,33 +5,35 @@
 import QtQuick 2.12
 import QtQuick.Controls 2.12
 import QtQuick.Layouts 1.11
-import "../components" as BitcoinCoreComponents
-
+import "../components"
 
 ApplicationWindow {
     id: appWindow
     title: "Bitcoin Core TnG"
     minimumWidth: 750
     minimumHeight: 450
```

Comment:
Do we have any plan on how to set the `minimumHeight` to correspond to the required minimum size of the contents?
<img width="862" alt="Screen Shot 2021-11-02 at 7 27 24 PM" src="https://user-images.githubusercontent.com/23396902/139965582-25901603-9537-4f5a-829f-154eb73b5668.png">

Reply:
We should set these once we settle down the main/window layout.